README.md

```diff
@@ -1,30 +1,39 @@
 # Conda documentation
 
-This repository includes documentation that is common for `conda` and `conda-build` (primarily landing pages at https://conda.io/).
+This repository includes documentation that is common for `conda` and `conda-build`, as well as landing pages for those projects (located at https://docs.conda.io/).
+
+The conda documentation is built using [ReadTheDocs](https://readthedocs.com/). The `conda-docs` repo is the primary project, while `conda` and `conda-build` are created as subprojects. This enables each project's respective documentation to remain in its repo, but for all of the documentation to exist together on the same domain. The landing pages built from the `conda-docs` repo link to the documentation built from the `conda` and `conda-build` repos.
 
 The documentation for each of those individual repositories can be found in their respective repos:
 
 - https://github.com/conda/conda
 - https://github.com/conda/conda-build
 
-Please file issues, comments, and PRs at the appropriate repo.
+Please file issues, comments, and pull requests at the appropriate repo.
+
+## Contribution guidelines
+
+### Conda capitalization
```

Comment:
I added this so the "creating new pages" guidance below wouldn't be all by itself. Anything else y'all think would be helpful for this section?

Reply:
This is useful, someone literally asked me the other day where to find it!